README.md

```diff
@@ -390,12 +390,13 @@ search.addWidget(
  * Display the list of results (hits) from the current search
  * @param  {String|DOMElement} options.container CSS Selector or DOMElement to insert the widget
  * @param  {Object} [options.templates] Templates to use for the widget
- * @param  {String|Function} [options.templates.empty=''] Template to use when there are no result
- * @param  {String|Function} [options.templates.hit=''] Template to use for each result
+ * @param  {String|Function} [options.templates.empty=''] Template to use when there are no results.
```

Comment:
Primitive types in JS are lower case. Maybe we should update the doc to reflect that. Also probably out of the scope of this PR.